games-simulation/openrct2/openrct2-0.1.0.ebuild

```diff
@@ -0,0 +1,99 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils gnome2-utils
+
+DESCRIPTION="An open source re-implementation of RollerCoaster Tycoon 2"
+HOMEPAGE="https://openrct2.website/"
+MY_REPO="https://github.com/OpenRCT2/OpenRCT2"
```

Comment:
I dare say this variable causes more confusion than repeating the URL, so please inline it.